react/react1/week1/react-week1/src/App.js

```diff
@@ -0,0 +1,20 @@
+const Todos = [
+  {id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" }, 
+  {id: 2, description: "Brush teeth", deadline: "Thu Sep 14 2017"}, 
+  {id: 3, description: "Eat breakfast", deadline: "Fri Sep 15 2017" }
+  // {id: 4, description: "Eat launch", deadline: "Fri Sep 16 2017" }
+];
```

Comment:
I would recommend to remove unnecessary default file from app. fx. public folder.  favicon.ico, logo192.png, logo512.png, manifest.jason , robots.txt. you can also remove file from src folder fx. logo.svg, reportWebVitals.js, setupTest.js and try to make index.html and index.css in public folder.